registry.go

```diff
@@ -1,9 +1,13 @@
 package shimgo
 
 import (
+	"fmt"
+	"io/ioutil"
+	"os"
 	"sync"
 )
 
+var workingDirectory string
```

Comment:
can we revert the change that requires having working directory be a package variable? I very much wanted to keep this encapsulated in the server implementation/internals. 

I think we can have one map of the service files without needing to have a global working directory (which makes this non-reentrant, in ways that I'd like to avoid.